Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/App.xaml

```diff
@@ -1,9 +1,8 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <Application xmlns="http://xamarin.com/schemas/2014/forms"
              xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
              x:Class="Contoso.Forms.Demo.App">
     <Application.Resources>
         <!-- Application resource dictionary -->
-        <Color x:Key="SwitchCellOnColor">Accent</Color>
```

Comment:
As far as I remember it was some bugfix, please revert or make sure that isn't required anymore

Reply:
Revert and add it to eventfilter toggle.